src/java.base/share/classes/java/util/concurrent/AbstractExecutorService.java

```diff
@@ -154,12 +155,11 @@ public <T> Future<T> submit(Callable<T> task) {
      */
     private <T> T doInvokeAny(Collection<? extends Callable<T>> tasks,
                               boolean timed, long nanos)
-        throws InterruptedException, ExecutionException, TimeoutException {
-        if (tasks == null)
-            throw new NullPointerException();
+        throws InterruptedException, ExecutionException, TimeoutException, NullPointerException {
```

Comment:
NullPointerException is a RuntimeException

```suggestion
        throws InterruptedException, ExecutionException, TimeoutException {
```